zh/access-dashboard.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 访问 TiDB Dashboard
+summary: 介绍如何在 Kubernetes 环境下访问 TiDB Dashboard
+category: how-to
+---
+
+# TiDB Dashboard 指南
```

Comment:
The level-1 title is not consistent with the title in metadata.

Reply:
Thanks. I'll fix it later.